src/pages/deploy/exposing-your-app.md

```diff
@@ -1,7 +1,15 @@
 ---
-title: Custom Domains
+title: Exposing Your App
 ---
 
+## Exposing Your Application
+
+Before your application can say hello, Railway needs to know what PORT to listen on to expose your application to the internet. Railway does try to do it's best to do this automatically for you however there are cases when we can't.
+
+You can configure your application to use the `PORT` environment variable. You can add the `PORT` variable under your project's variables. (Command + K and type `Variables` or you can use the keyboard shortcut: `G` + `V` under your selected project)
```

Comment:
Not super happy with my suggestion, but previously I felt it was a bit wordy and used the word "variable" too many times. Feel free to change

```suggestion
You can configure your application to use the `PORT` environment variable by adding the `PORT` on your projects variables page. (Command + K and type `Variables` or you can use the keyboard shortcut: `G` + `V` under your selected project)
```